composer.json

```diff
@@ -13,7 +13,7 @@
     },
     "minimum-stability": "stable",
     "require": {
-        "php": ">=8.0",
+        "php": ">=8.1",
```

Comment:
An increase in the minimum PHP version is a little more substantial and should also be coordinated first. 
See staff notes here: https://staff.humhub.org/s/development/wiki/146/increase-min-php-version

In this case, however, it works, as this can be planned in v.17.